src/Entity/PhoneNumberSettings.php

```diff
@@ -56,7 +56,7 @@ class PhoneNumberSettings extends ConfigEntityBase implements PhoneNumberSetting
    *
    * @var string
    */
-  protected $entity_type;
+  protected $entityType;
```

Comment:
i think there is a misunderstanding about the purpose of the properties entities. Config properties are automatically populated based on *schema.yml. Which is why you see code like    

```yaml
verification_message:
       type: string
```

to an entity.php:

```php
class MyConfigEntity {
 -  protected $verification_message; 
}
```